app/models/blood_pressure.rb

```diff
@@ -18,6 +18,7 @@ class BloodPressure < ApplicationRecord
 
   scope :hypertensive, -> { where("systolic >= 140 OR diastolic >= 90") }
   scope :under_control, -> { where("systolic < 140 AND diastolic < 90") }
+  scope :with_encounters, -> { includes(:encounter).where.not(encounters: { id: nil }) }
```

Comment:
Looks like this new scope isn't being used anywhere else in the PR. Can we remove?